consensus/config/src/parameters.rs

```diff
@@ -45,6 +45,13 @@ pub struct Parameters {
     #[serde(default = "Parameters::default_sync_last_known_own_block_timeout")]
     pub sync_last_known_own_block_timeout: Duration,
 
+    /// The number of rounds above the highest accepted round that still willing to fetch missing blocks via the periodic
+    /// synchronizer. Any missing blocks of higher rounds are considered too far in the future to fetch. This property is taken into
+    /// account only when it's detected that the node has fallen behind on its commit compared to the rest of the network, otherwise
+    /// scheduler will attempt to fetch any missing block.
+    #[serde(default = "Parameters::default_sync_missing_blocks_round_threshold")]
+    pub sync_missing_blocks_round_threshold: u32,
```

Comment:
I can see having separate parameters can be conceptually cleaner. But this parameter seems to have less room for adjustment, compared to for example timeouts. Not sure if we want to expose too many parameters either. How about just reusing gc_rounds and something else when gc_rounds is 0?

Reply:
Why you think this parameter has less room for adjustment?  Indeed it would be nice to reuse some of the existing ones but it's a bit hard for me to find some clear correlation which makes sense without some implicit semantics. Also I believe if I tight the mechanism to some other parameter, especially critical for safety (ex gc_depth) , then we might not be able to adjust easily if needed.

Reply:
I guess I mean less need for adjustments. In theory, we just need 1 additional round for liveness, right?

Reply:
I believe we don't even need 1 additional round for liveness, we could just use the `highest_accepted_round`. But it might be good to have some threshold anyways even for efficiency reasons. Ok, what if instead of a config property I just use an in-code constant for now (originally had it like this but then thought that a property config would be more versatile)?

Reply:
Yes I think a const is definitely better.